smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -254,6 +254,7 @@ private String getCommandExample(String serviceName, String configName, String c
                         packageName)
                 + String.format("// const { %s, %s } = require(\"%s\"); // CommonJS import%n", serviceName, commandName,
                         packageName)
+                + "// See AWS SDK config options: https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/configuring-the-jssdk.html\n"
                 + String.format("const client = new %s(config);%n", serviceName)
```

Comment:
this is smithy, we shouldn't link to AWS docs.

A preface block to the line containing the client init should be injectable by the downstream code generator. You can refer to the `CodeSection` interface usage in this repo on how to do that.